tripleo-quickstart-config/metalkube-nodes.yml

```diff
@@ -1,4 +1,4 @@
-default_memory: 8192
+default_memory: 16384
```

Comment:
In general, I think we should match defaults used on other supported platforms.

That's 16 GB for masters, but 8 GB for workers.  We could save a bit by defaulting to 8 GB for the worker.